File-Formats/PDF/Convert-HTML-To-PDF/azure.md

```diff
@@ -174,139 +162,134 @@ A complete work sample for converting an HTML to PDF in Azure App service on Lin
 
 ### Steps to convert HTML to PDF in Azure app service using the Blink with Linux docker container:
 
-Create a new ASP.NET Core application and enable the docker support with Linux as a target OS.
+1. Create a new ASP.NET Core application and enable the docker support with Linux as a target OS.
 ![Convert HTMLToPDF Azure Docker Step1](htmlconversion_images/DockerStep1.png)
+
+2. Choose your project's target framework, then select Configure for HTTPS and Enable Docker.
```

Comment:
Modify as "Choose your project's target framework and select Configure for HTTPS and Enable Docker."

Reply:
Modified